pkg/apis/eventing/v1beta1/eventtype_defaults.go

```diff
@@ -28,9 +28,6 @@ func (et *EventType) SetDefaults(ctx context.Context) {
 }
 
 func (ets *EventTypeSpec) SetDefaults(ctx context.Context) {
-	if ets.Reference == nil && ets.Broker == "" {
-		ets.Broker = "default"
```

Comment:
I guess for newly created `v1b1`, this is not that bad. No need for some post migration logic.